src/Snipe.php

```diff
@@ -82,7 +89,7 @@ protected function migrationFileTimeSum()
                     ->sum(function ($file) {
                         return $file->getMTime();
                     });
-            })->sum();
+            })->sum() + (config('snipe.seed-database', false) ? 1 : -1);
```

Comment:
This was added because now it's possible that the migrations don't change, but the seed preference does. Changing the seed preference will invalidate the file time cache, and cause the DB to be re-migrated with or without seeding.

Reply:
thanks @JackWH. I'll take a look later today or tomorrow and merge it.

Reply:
@drfraker Hi! Just wondered if you were happy to merge this yet? Would be great for my teammates to be able to use this change in our project.

Reply:
@JackWH Will making changes to the seed file cause the migrations to re-run automatically, or would a user be required to clear the snapshot if they change the seeder?